curations/maven/mavencentral/org.webjars.bowergithub.webcomponents/shadycss.yaml

```diff
@@ -0,0 +1,15 @@
+coordinates:
+  name: shadycss
+  namespace: org.webjars.bowergithub.webcomponents
+  provider: mavencentral
+  type: maven
+revisions:
+  1.5.0-1:
+    described:
+      sourceLocation:
+        name: shadycss
+        namespace: org.webjars.bowergithub.webcomponents
```

Comment:
This `namespace` should `webcomponents` since this is for the `sourceLocation`

Reply:
Interesting, so in case of a GitHub source location the namespace should be the GitHub organization / user name?

Reply:
Yes :-)

Reply:
Here is an entry of a GitHub sourceLocation: https://github.com/clearlydefined/curated-data/blob/3ffd7dfd3d57736462b233287845e54e95acaa26/curations/nuget/nuget/-/Microsoft.IdentityModel.Clients.ActiveDirectory.yaml#L47

Reply:
So the URL should also point to the commit, not to the project page?

Reply:
> This `namespace` should `webcomponents` since this is for the `sourceLocation`

Done (in new PR).